docs/platform/enterprise-flex/data-plane.md

```diff
@@ -0,0 +1,452 @@
+---
+products: enterprise-flex
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Multiple region deployments with Enterprise Flex
```

Comment:
What do you think of splitting this into two docs: 
- Getting Started
- Self-Managed Regions

The idea of the `Getting Started doc` is to give a lot of the high level context included at the top here, and to outline the `ease of use / cloud region` side of the setup. Also, it's to show that if you just follow the getting started guide on it's own, you can start running syncs.

The goal is that the guide is very short, and then you can follow on from the second page on `Self-Managed Regions`